DAL/Repositories/PaymentRepository.cs

```diff
@@ -21,16 +21,19 @@ public PaymentRepository(PaymentsDbContext context)
 
         public async Task<TransactionDTO> CreateTransaction(TransactionDTO transaction)
         {
+
             await _context.Transactions.AddAsync(transaction);
-            _context.SaveChanges();
+            await _context.SaveChangesAsync();
 
             return transaction;
         }
 
         public async Task<IEnumerable<TransactionDTO>> CreateTransaction(IEnumerable<TransactionDTO> transaction)
```

Comment:
Seems it should be a plural form used here

Reply:
I will change it